blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""This module contains models for the blog app."""
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Model representing a blog post."""
+
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
```

Comment:
Why do you need it both blank and null? Is that what the tutorial suggests?

Reply:
Yes, we have this example in the tutorial.

Reply:
Marking as resolved.